stdlib/3/ssl.pyi

```diff
@@ -207,7 +207,10 @@ class SSLContext:
     if sys.version_info >= (3, 4):
         verify_flags = ...  # type: int
     verify_mode = ...  # type: int
-    def __init__(self, protocol: int) -> None: ...
+    if sys.version_info >= (3, 5):
+        def __init__(self, protocol: Optional[int]) -> None: ...
```

Comment:
Do you mean `protocol: int = ...`? This doesn't set a default value; it just allows None to be passed.

Reply:
Silly me... Fixed